pdr_backend/sim/sim_engine.py

```diff
@@ -140,12 +148,17 @@ def run_one_iter(self, test_i: int, mergedohlcv_df: pl.DataFrame):
 
         curprice = yraw[-2]
         trueprice = yraw[-1]
-
+        shifted_mergedohlcv_df = mergedohlcv_df[-testshift - 2]
+        high_col = f"{predict_feed.exchange}:{predict_feed.pair}:high"
+        low_col = f"{predict_feed.exchange}:{predict_feed.pair}:low"
+        high_value = shifted_mergedohlcv_df[high_col].to_numpy()[0]
+        low_value = shifted_mergedohlcv_df[low_col].to_numpy()[0]
```

Comment:
Please turn this into a one-line call to data_f.get_highlow(predict_feed, mergedohlcv_df). Then move it to right before the line `profit = self.sim_trader`.

And implement get_highlow(), of course.